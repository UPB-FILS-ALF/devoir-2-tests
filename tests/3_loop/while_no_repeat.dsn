& while no repeat
SET: %var 200

WHILE: %var
COLOR:  fill 200 10 30
CIRCLE: 60 200 200 
COLOR:  pen 10 40 250
LINE:  100 150 polar
IDIV: %var 100

IF: %var
ELLIPSE: 100 20 400 400 
POSITION: 300 300
COLOR:  pen 0 40 0
LINE:  700 500 location
END