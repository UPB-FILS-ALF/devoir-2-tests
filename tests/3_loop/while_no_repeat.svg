<?xml version="1.0" encoding="UTF-8" ?>
<svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="1024" height="768">
	<!-- var = 200 -->
	<!-- while var: 200 -->
	<!-- while #0 var: 200 -->
		<!-- fill 200, 10, 30 -->
		<circle cx="60" cy="200" r="200" stroke="rgb(0,0,0)"  stroke-width="1" fill="rgb(200,10,30)"/>
		<!-- pen 10, 40, 250 -->
		<line x1="0" y1="0" x2="-87" y2="-50" stroke="rgb(10,40,250)"  stroke-width="1" />
		<!-- var = var / 100 : 2 -->
		<!-- if var: 2 -->
			<ellipse cx="100" cy="20" rx="400" ry="400" stroke="rgb(10,40,250)"  stroke-width="1" fill="rgb(200,10,30)"/>
			<!-- position 300, 300 -->
			<!-- pen 0, 40, 0 -->
			<line x1="300" y1="300" x2="700" y2="500" stroke="rgb(0,40,0)"  stroke-width="1" />
			<!-- end -->
</svg>
