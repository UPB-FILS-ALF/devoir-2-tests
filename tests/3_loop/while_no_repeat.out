LINE 4 HAS ERROR: WHILE with no REPEAT
